cmd/webhook/app/webhook_test.go

```diff
@@ -85,6 +85,15 @@ func TestFlagsAndConfigFile(t *testing.T) {
 	}
 
 	tests := []testCase{
+		{
+			yaml: ``,
+			args: func(tempFilePath string) []string {
+				return []string{"--kubeconfig=valid"}
+			},
+			expConfig: configFromDefaults(func(tempDir string, cc *config.WebhookConfiguration) {
+				cc.KubeConfig = "valid"
+			}),
+		},
```

Comment:
Again, I don't understand how **this** new test relates to the logging changes.

If anything, I expected to see you **remove** a test that demonstrated the previous behaviour. 
E.g. a test that shows configfile parsing errors can be logged using `json` output format.

Reply:
This testcase tests the case in which the `newConfigHook` function is only called once.

We did not have a test for checking the json output format.